controller/volume_controller.go

```diff
@@ -350,6 +350,28 @@ func (c *VolumeController) syncVolume(key string) (err error) {
 				}
 			}
 		}
+		if datastore.IsDataEngineV2(volume.Spec.DataEngine) {
+			// To prevent from the "no such device" error in spdk_tgt,
+			// remove the raid bdev before tearing down the replicas.
+			engines, err := c.ds.ListVolumeEnginesRO(volume.Name)
```

Comment:
Should we consider the engine's DeletionTimestamp?

Reply:
Although `engine.Spec.DeletionTimestamp` is set, it does not mean the engine (raid bdev) is already tear down. Hence, I only check the existence of engines here.
Do you think there is any side effect?

Reply:
I see. No concerns.